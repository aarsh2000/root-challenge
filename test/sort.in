Driver B
Trip B 00:10 00:50 43.25
Driver C
Trip C 10:34 11:12 34.5
Trip B 00:50 01:34 5
Driver A
Trip A 03:43 04:00 24.1
Trip C 21:00 23:00 17.4
