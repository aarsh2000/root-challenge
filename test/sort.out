C: 52 miles @ 20 mph
B: 48 miles @ 34 mph
A: 24 miles @ 85 mph
